syncapi/storage/postgres/deltas/20201017140347_create_receipt_table.go

```diff
@@ -0,0 +1,46 @@
+package deltas
```

Comment:
You don't need to write a delta for brand new tables. The code will CREATE TABLE IF NOT EXISTS.

Reply:
So just remove these delta files.